lib/awestruct/context_helper.rb

```diff
@@ -40,10 +40,10 @@ def summarize(text, numwords=20, ellipsis='...')
 
     def fully_qualify_urls(base_url, text)
       begin
-        doc = Oga.parse_html text
```

Comment:
Seems okay, but why?

Reply:
These are the failures I have if I run `rake build` with master.

```
Failures:                                                                                                                                
                                                                                                                                  
  1) Awestruct::ContextHelper fully_qualify_urls should fix image tags                                                                   
     Failure/Error: @tester.fully_qualify_urls('http://foobar.com', str).should == %q(<img src="http://foobar.com/foo" />)        
                                                                                                                                            
       expected: "<img src=\"http://foobar.com/foo\" />"                                                                          
            got: "<img src=\"http://foobar.com/foo\">" (using ==)                                                                        
     # ./spec/awestruct/context_helper_spec.rb:89:in `block (3 levels) in <top (required)>'                                                 
                                                                                                                                            
  2) Awestruct::ContextHelper fully_qualify_urls should fix link tags                                                               
     Failure/Error: @tester.fully_qualify_urls('http://foobar.com', str).should == %q(<link href="http://foobar.com/foo" />)                                                           
                                                                                                                                                                          
       expected: "<link href=\"http://foobar.com/foo\" />"                                                                                                                                                                                                                                                                                                                                    
            got: "<link href=\"http://foobar.com/foo\">" (using ==)                                                                 
     # ./spec/awestruct/context_helper_spec.rb:84:in `block (3 levels) in <top (required)>'     
```

Between Oga 1.3 and 2 something has changed in the way self_closing tag are managed.
In 1.3 `parse_html` add the `/` at the closing tag for `link` and `img`, so `<link href='/foo'>`
becomes `<link href="/foo" />`.

In Oga 2 the `/>` is transformed into `>`, that's because in html for link and img you don't need to close the tag.  Therefore  `<link href='/foo'>` becomes  `<link href="/foo">` and   `<img src='/foo' />` becomes `<img src="/foo" >`. it seems that parse_xml always closes that tags and from the tests I've seen it looks like the expected behavior. 

I would have expected to have a way to specify the header so that one can say if the xml is an html or xhtml. But I couldn't find a way to do that.

Anyway, I'm not familiar with Ruby or Oga so I'm not sure if there is some reasoning behind this change in behavior.

The failures have been introduced by commit: 7dc3b91